database/mock_data/PopulateDB_Part2.sql

```diff
@@ -1,14 +1,14 @@
 ### GEN THE JOBS
-CALL sp_event_job_generation('2020-02-01T08:00:00', '2020-02-02T16:00:00', 1, 'The First Race', 'test first race!', @ignore);
```

Comment:
If you change these back to 2020, the integration tests should pass. Looks like this got lost in the merge.